pkg/kubelet/rkt/rkt.go

```diff
@@ -154,13 +155,22 @@ func New(config *Config,
 	if err != nil {
 		return nil, err
 	}
-	result, err = version.Compare(rktMinimumVersion)
+	result, err = version.Compare(minimumRktVersion)
 	if err != nil {
 		return nil, err
 	}
 	if result < 0 {
-		return nil, fmt.Errorf("rkt: Version is too old, requires at least %v", rktMinimumVersion)
+		return nil, fmt.Errorf("rkt: version is too old, requires at least %v", minimumRktVersion)
 	}
+
+	result, err = version.Compare(recommendRktVersion)
+	if err != nil {
+		return nil, err
+	}
+	if result != 0 {
+		glog.Warningf("rkt: current version %q is not recommended (recommended version %q)", version, recommendRktVersion)
```

Comment:
I think we'd want to record an event or surface this through in other ways, but we can reiterate later.